py/desisim/io.py

```diff
@@ -1144,7 +1144,7 @@ def empty_metatable(nmodel=1, objtype='ELG', subtype='', simqso=False, input_met
     if input_meta:
         meta.add_column(Column(name='TEMPLATEID', length=nmodel, dtype='i2', data=np.zeros(nmodel)-1))
         meta.add_column(Column(name='SEED', length=nmodel, dtype='int64', data=np.zeros(nmodel)-1))
-        meta.add_column(Column(name='REDSHIFT', length=nmodel, dtype='f4', data=np.zeros(nmodel)))
+        meta.add_column(Column(name='REDSHIFT', length=nmodel, dtype='f8', data=np.zeros(nmodel)))
```

Comment:
Is this consistent with redshift values output by the real pipeline?  In general what is the advantage of using a `double` when the real precision is much, much less than that?

Reply:
Yes, the real pipeline outputs double-precision redshifts. I made this change in order to be consistent.